### PR TITLE
Roadmap Update for CNI genie

### PR DESCRIPTION
**Pull request by sushanthakumar**
Roadmap Update for CNI genie

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## Openness
+ - Enhancements as per Network Plumbing Group conclusions/decisions
+ - CNI version upgrade based on new CNI version release
+ - Support pod level network policy to co-exist with network level policy
+ - Enhancement of network crd objects to provide more CNI customizations
+ - Enhance network smart selection mechanism
+ - New requirement/usecase support based on users demands
+
+## User experience
+ - Integrate genie with other eco system projects (ex: kubespray, kubevirt)
```
Comment (m1093782566) [inline ROADMAP.md]: ecosystem?
and virtlet?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# CNI-Genie Roadmap
+
+## Openness
+ - Enhancements as per Network Plumbing Group conclusions/decisions
+ - CNI version upgrade based on new CNI version release
+ - Support pod level network policy to co-exist with network level policy
+ - Enhancement of network crd objects to provide more CNI customizations
+ - Enhance network smart selection mechanism
+ - New requirement/usecase support based on users demands
```
Comment (m1093782566) [inline ROADMAP.md]: And SR-IOV support?

File: ROADMAP.md (hunk 1 above)
Comment (kevin-wangzefeng) [reply ROADMAP.md]: s/ex:/e.g./

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# CNI-Genie Roadmap
+
+## Openness
+ - Enhancements as per Network Plumbing Group conclusions/decisions
```
Comment (kevin-wangzefeng) [inline ROADMAP.md]: s/Network Plumbing Group/Kubernetes Network Plumbing Working Group/

**Pull request merged**